pyInterface/storageFormats/amplitudeTreeLeaf_py.cc

```diff
@@ -105,12 +87,12 @@ void rpwa::py::exportAmplitudeTreeLeaf() {
 		.def("setAmp", &rpwa::amplitudeTreeLeaf::setAmp)
 
 		.def("Write", &amplitudeTreeLeaf_Write, bp::arg("name")=0)
-		.def("setBranchAddress", &amplitudeTreeLeaf_setBranchAddress)
+		.def("setBranchAddress", &rpwa::py::setBranchAddress<rpwa::amplitudeTreeLeaf*>)
 		.def(
 			"branch"
-			, &amplitudeTreeLeaf_branch
-			, (bp::arg("pyTree"),
-			   bp::arg("pyAmplitudeTreeLeaf"),
+			, &rpwa::py::branch<rpwa::amplitudeTreeLeaf*>
+			, (bp::arg("amplitudeTreeLeaf"),
+			   bp::arg("tree"),
 			   bp::arg("name"),
```

Comment:
The indentation and newlines differ to those changed above in `fitResult_py.cc`, this should probably be made consistent.